chapters/chap02-sec03.tex

```diff
@@ -49,3 +49,11 @@ \section{Generating the inputs of the $\mix$ function (\inp{\mix})}\label{zeth-p
     \end{enumerate}
     Here, $\eparty{S}.\addr$ represents the address of the $\ethereum{}$ user $\eparty{S}$ who must sign the transaction (see~\cref{zeth-protocol:create-tx}). In general, this is likely to be owned by the holder $\zparty{U}$ of the $\zeth{}$ notes to be spent, but this is not a requirement.
 \end{enumerate}
+
+\begin{remark}\label{zeth-protocol:mix-inp:otsig-for-dispatch-call}
+  In some specific cases (where the proof $\inp{\mix}.\zkp$ will be verified by some trusted external mechanism), as described in \cref{zeth-protocol:process-tx:dispatch-call}a, it may be necessary to replace $\datatobesigned$ in \cref{zeth-protocol:mix-inp:create-otsig} with:
```

Comment:
I'll update.  I can see where the confusion may arise, given the way "trustless" is often used, but worth noting that "trust" does not necessarily mean "in the absence of evidence".  The given party "trusts" because he is sufficiently convinced. (Naturally, it would not make sense to require that any party believe in the reliability of the external verifier on faith alone.)

Reply:
Updated this comment.  I've stripped it back, but tbh I was considering simply deleting this remark. It is a bit perpendicular to the specification, so not necessarily worth stating twice.